### ISSUE TITLE
[Content] SC Step 3- Alert on Review screen

### ISSUE THREAD

**Issue by kbermangestring** [department-of-veterans-affairs/va.gov-team#131668]
## Issue
If a claimant does not add VA providers, private providers or upload new documents they will recieve an alert that they have not submitted new evidence. 

New evidence is required for supplemental claims, however for presumptive conditions it is not. We need to alert the Veteran that this may be required but not block them from submitting their supplemental claim. 

## Task
- [ ] Review content [currently in production](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims--VA-0995-?node-id=8805-57844&t=NY3fUATF8U97QhXA-11) and update if necessary. 
- [ ] Handoff to A11y for review & handoff to eng.

## Acceptance Criteria
- [ ] All tasks have been completed.

**Comment by cinlackey** [comment]
[Reviewed and proposed new content for the alert](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims--VA-0995-?node-id=8805-57844&t=bpfDBLv6ofwSnRbe-0)
Outstanding question to UX team and/or Julie on [presumptive conditions re: evidence](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A?node-id=8805-57844#1612458233)
